lib/bsdlib/Kconfig

```diff
@@ -32,6 +32,24 @@ config BSD_LIBRARY_TRACE_ENABLED
 	# This enable UARTE1 peripheral and includes nrfx UARTE driver.
 	select NRFX_UARTE1
 
+config NRF91_SOCKET_SEND_SPLIT_LARGE_BLOCKS
+	bool "Split large blocks passed to send() or sendto()"
+	default n
+	help
+	  Workaround a limitation in the nRF91 modem firmware's bsdlib
+	  implementation regarding return value for send() or sendto()
+	  calls larger than the module can handle. It should send the data
+	  up to the maximum, and return that as the return value. Instead,
+	  it returns error 22.
+
+config NRF91_SOCKET_BLOCK_LIMIT
```

Comment:
Should it be a configurable value? I guess the limit on the bsdlib side is "fixed"?